lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -202,15 +213,19 @@ export function createBucket(authInfo, bucketName, headers,
             return cb(errors.BucketAlreadyExists);
         }
         const newBucketMD = results.prepareNewBucketMD;
-        const serverSideEncryption = results.serverSideEncryption;
-        if (serverSideEncryption) {
-            newBucketMD.setServerSideEncryption(serverSideEncryption);
-        }
         if (existingBucketMD === 'NoBucketYet') {
             log.trace('new bucket without flags; adding transient label');
-            newBucketMD.addTransientFlag();
```

Comment:
I'd prefer to keep the log about the flag right before adding the flag so please either move the log down to below or move the newBucketMD.addTransientFlag() back to up here.